### PR TITLE
Change TZNG examples/refs to ZSON

### PR DESCRIPTION
**Pull request by philrz**
Updating the user-facing docs updates in #2294 made me aware of other lingering TZNG references in the docs that could also stand to be updated to ZSON. While I was in there, I found a few other links and missing pieces that seemed worth adding.

The zst spec was already pretty drafty so I wasn't 100% sure if the update to ZSON there was appropriate. But the fact I could actually use the comments in the ZSON to document the example in-line felt like enough of an improvement to justify it. I'm not sure it's enough to close the issues #1714 and #1734 (which appear to be duplicates of each other, FWIW) but since zst is his baby I'll wait to see what @mccanne has to say.

### CODE REVIEW

**Review by mccanne — APPROVED** [review]
cool

File: cmd/zq/README.md [hunk 1]
```diff
@@ -20,21 +20,21 @@ data, which is used in the examples in the
...
- terminal, use `cut`:
+ terminal, use [`cut`](../../zql/docs/processors/README.md#cut):
 
 ```
-zq -t "* | cut ts,id.orig_h,id.orig_p" conn.log
+zq -z "* | cut ts,id.orig_h,id.orig_p" conn.log
 ```
 
-The `-t` tells `zq` to use [TZNG](../../zng/docs/spec.md#4-zng-text-format-tzng)
+The `-z` tells `zq` to use human-readable text [ZSON](../../zng/docs/zson.md)
```
Comment (nwt) [inline cmd/zq/README.md:29]: While they certainly don't mean the same thing, in this context "text" feels redundant after "human-readable".
```suggestion
The `-z` tells `zq` to use human-readable [ZSON](../../zng/docs/zson.md)
```

File: zst/README.md [hunk 2]
```diff
@@ -449,13 +449,19 @@ using the same selector value within the union value.
 
 ### Hello, world
 
+Start with this zng data (printed in human-readable [ZSON](../zng/docs/zson.md)
+text format):
```
Comment (nwt) [inline zst/README.md:453]:
```suggestion
Start with this zng data (shown as human-readable [ZSON](../zng/docs/zson.md)):
```

File: zst/README.md [hunk 3]
```diff
@@ -468,83 +474,58 @@ gracie
...
-zst inspect -f ndjson -trailer hello.zst | jq
-```
-to view it all a bit more easily (with loss of information due to json):
+which provides the output (comments added with explanations):
+
 ```
 {
-  "a": null,
-  "b": null
+    a: null (string),                // First, the schemas are defined (just one here)
```
Comment (nwt) [inline zst/README.md:488]:
```suggestion
    a: null (string),                // First, the schemas are defined (just one here).
```

File: zst/README.md [hunk 4]
```diff
@@ -468,83 +474,58 @@ gracie
...
 }
 {
-  "root": [
-    {
-      "length": 2,
-      "offset": 29
-    }
-  ]
-}
+    root: [                          // Then, the root reassembly record
```
Comment (nwt) [inline zst/README.md:492]:
```suggestion
    root: [                          // Then, the root reassembly record.
```

File: zst/README.md [hunk 5]
```diff
@@ -468,83 +474,58 @@ gracie
...
-      {
-        "length": 13,
-        "offset": 16
-      }
-    ],
-    "presence": []
-  }
-}
+    a: {                             // Next comes the column assembly records
+        column: [                    // (again, only one schema in this example, so only one such record)
```
Comment (nwt) [inline zst/README.md:501]:
```suggestion
    a: {                             // Next comes the column assembly records.
        column: [                    // (Again, only one schema in this example, so only one such record.)
```

File: zst/README.md [hunk 6]
```diff
@@ -468,83 +474,58 @@ gracie
...
-  "magic": "zst",
-  "sections": [
-    31,
-    94
-  ],
-  "segment_thresh": 5242880,
-  "skew_thresh": 26214400,
-  "version": 1
-}
+    magic: "zst",                    // Finally, the trailer as a new zng stream
```
Comment (nwt) [inline zst/README.md:520]:
```suggestion
    magic: "zst",                    // Finally, the trailer as a new zng stream.
```

**Review by nwt — APPROVED** [review]

**Pull request merged**